backend/useragent/user_agent.go

```diff
@@ -0,0 +1,50 @@
+package useragent
```

Comment:
maybe we can use an external library for this? https://github.com/mileusna/useragent

That also helps with using well-known formats

Reply:
It's not really this sort of client information we're looking for - it's more the Grafana backend information (as the frontend stuff is really relevant)

Reply:
I think it's a bit overkill for what we need considering we will control exactly what format it takes, especially what that lib seems geared around browser + bot user agents. I also dislike the idea of having a third party API as part of the SDK API - rolling out own gives us control over what to expose, rather than the large list of methods like `IsInternetExplorer`, `IsIOS`, etc. which aren't relevant for us.

Regexs may look scary but this one is fairly trivial 😇

Reply:
:shrug: not feeling strongly about this, if you want to still keep it then go for it